ruby/vampires.rb

```diff
@@ -0,0 +1,98 @@
+#Driver code
+#Process multiple employees
+puts "How many employees will be processed?"
+ee_num = gets.chomp.to_i
+#initiating loop to process multi employees. It wraps code
+count = 0
+while count < ee_num
```

Comment:
Look into using the Integer#times iteration

http://ruby-doc.org/core-2.2.0/Integer.html#method-i-times